db/seeds.d/07-provisioning_templates.rb

```diff
@@ -1,14 +1,14 @@
 # Find known operating systems for associations
-os_junos = Operatingsystem.find_all_by_type "Junos" || Operatingsystem.where("name LIKE ?", "junos")
-os_solaris = Operatingsystem.find_all_by_type "Solaris"
-os_suse = Operatingsystem.find_all_by_type "Suse" || Operatingsystem.where("name LIKE ?", "suse")
-os_windows = Operatingsystem.find_all_by_type "Windows"
+os_junos = [Operatingsystem.where(:type => "Junos").first || Operatingsystem.where("name LIKE ?", "junos").first].compact
```

Comment:
I don't think these should be `[...first]`, they should simply be `Operatingsystem.where(:type => "Junos")`